### PR TITLE
User can edit business from business details page

### PR DESCRIPTION
**Pull request by jasonramirez**
https://headquarters.io/tinysplash/stories/937

### CODE REVIEW

File: app/controllers/businesses_controller.rb [hunk 1]
```diff
@@ -20,6 +20,16 @@ def create
     end
   end
 
+  def edit
+    @business = Business.friendly.find(params[:id])
+  end
+
+  def update
+    business = Business.friendly.find(params[:id])
+    business.update(business_params)
```
Comment (dgalarza) [inline app/controllers/businesses_controller.rb]: This probably should take into account any validation errors that might occur.

How about:

```ruby
def update
  business = Business.friendly.find(params[:id])
  
  if business.update(business_params)
    redirect ...
  else
    render :edit
  end
end
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -21,6 +23,8 @@ en:
         create: "Submit"
       fake_door_campaign:
         create: "Create Campaign"
+      business:
```
Comment (dgalarza) [inline config/locales/en.yml]: :abc:

File: spec/features/user_can_edit_a_business_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    click_link t("businesses.show.edit")
+    fill_form_and_submit(
+      :business,
+      :edit,
+      name: "Newer business",
+      mission: "To win"
+    )
+
+    expect(page).to have_content "Newer business"
```
Comment (dgalarza) [inline spec/features/user_can_edit_a_business_spec.rb]: Is there a flash notice we might want to check for as well?

**Comment by dgalarza** [comment]
Couple small comments, otherwise :hamburger:

File: spec/features/user_can_edit_a_business_spec.rb (hunk 3 above)
Comment (jasonramirez) [inline spec/features/user_can_edit_a_business_spec.rb]: sure

**Pull request merged**